src/components/Games/Number/NumberMain.js

```diff
@@ -47,28 +60,43 @@ const Input = styled.input`
 `;
 
 export const NumberMain = () => {
-  const [num, setNum] = useState(0);
+  const [num, setNum] = useState(Math.floor(Math.random() *  (9 - 1 +1)) +1);
+  const [newNum, setNewNum] = useState(Math.floor(Math.random() *  (9 - 1 +1)) +1);
 
   function RandomNumberInRange(min, max) {
```

Comment:
Move this function outside of the component as it's more of a utility and doesn't require access to the state.